plugins/source/gcp/client/client.go

```diff
@@ -19,9 +19,10 @@ const maxProjectIdsToLog int = 100
 
 type Client struct {
 	// plugin   *plugins.SourcePlugin
-	projects []string
+	projects      []string
+	ClientOptions []option.ClientOption
 	// All gcp services initialized by client
-	Services *Services
+	// Services *Services
```

Comment:
Nit: let's completely remove this

Reply:
Is the `Services` struct still used anywhere after this? Can it be removed? (I don't have the code checked out locally right now to check)